File: April-18/11-04-18/python/01-Prime-Number.py
prime = []
def seive():
    check = [0]*1000;
    for i in range(2, 1000):
        if(check[i] == 0):
            prime.append(i)
            for j in range(2*i,1000,i):
                check[j] = 1
seive()
n = int(input())
for val in prime:
    if(val > n):
        break
    print(val, end = ' ')

File: April-18/11-04-18/java/01-Prime-Number.java

import java.util.*;

class TestClass {
    static Vector vec = new Vector();
    static void seive()
    {
        int check[] = new int[1001];
        for(int i = 2; i <= 1000; i++)
        {
            if(check[i] == 0)
            {
                vec.add(i);
                for(int j = 2*i; j <= 1000; j += i)
                {
                    check[j] = 1;
                }
            }
        }
    }
    public static void main(String args[] ) throws Exception {
        seive();
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        Enumeration vEnum = vec.elements();
        int tmp = n;
        while(vEnum.hasMoreElements())
        {   
            int pri = (int)vEnum.nextElement();
            if(pri > n)
            {
                break;
            }
            System.out.print(pri + " ");
            
        }
    }
}
